api/src/test/java/org/openmrs/util/DatabaseUpdaterDatabaseIT.java

```diff
@@ -29,7 +29,7 @@
 	 * 
 	 * This constant needs to be updated when adding new Liquibase update files to openmrs-core.
 	 */
-	private static final int CHANGE_SET_COUNT_FOR_2_1_X = 863;
+	private static final int CHANGE_SET_COUNT_FOR_2_1_X = 864;
```

Comment:
What is in core is already 864. Can you increment this again?

Reply:
@dkayiwa  I have incremented